CHANGELOG.md

```diff
@@ -1,24 +1,37 @@
 # Changelog
 
-## 1.6.0 (TBD)
+## 1.7.0 (TBD)
+
+### Breaking changes
+
+- **Title** Description
+
+### Infrastructure breaking changes
+
+- **Title** Description
 
 ### New features
 
-- Certificate handlebar for registration fees `registrationFees` [#6817](https://github.com/opencrvs/opencrvs-core/issues/6817)
-- Logged in user details handlebar `loggedInUser` [#6529](https://github.com/opencrvs/opencrvs-core/issues/6529)
-- Supporting document fields can now be made required
-- If there is only one option in the document uploader select, then it stays hidden and only the upload button is showed with the only option being selected by default
+- **Major new feature** Description
+- Misc new feature
 
-- #### ElasticSearch reindexing
+### New content keys requiring translation
```

Comment:
If in the upgrade instructions we guided the team to run `yarn extract:translations`, then I think they would get this same list as the output of the script and we wouldn't have to manually maintain it.

https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/extract-translations.ts#L192

There's also the `--write` option that automatically adds the items in the country config. How do we integrate this better to the upgrade process?